src/main/java/com/uid2/shared/secure/TrustedAttestationProvider.java

```diff
@@ -12,7 +12,7 @@ public TrustedAttestationProvider() {}
 
     @Override
     public void attest(byte[] attestationRequest, byte[] publicKey, Handler<AsyncResult<AttestationResult>> handler) {
-        handler.handle(Future.succeededFuture(new AttestationResult(publicKey)));
+        handler.handle(Future.succeededFuture(new AttestationResult(publicKey, "Trusted")));
```

Comment:
How do we define the enclave ID?

Reply:
The enclave Id is dependent on the type of attestation and is returned by the Attestation Provider. For trusted operators, there is no attestation process, so no enclave id.